browser/src/layer/vector/SVGGroup.js

```diff
@@ -9,6 +9,20 @@ L.SVGGroup = L.Layer.extend({
 
 	options: {
 		noClip: true,
+		manualDrag: window.touch.isTouchEvent,
+	},
+
+	_noManualDragMemo: {},
+	noManualDrag: function(f) {
```

Comment:
can we create Util with this functionality?
I see it is reapeated in [Draggable.js](https://github.com/CollaboraOnline/online/pull/7712/files#diff-67a550f117b7181778af814172e9a1ad841e2c15bcc0b98568724284eb44fc7c)